src/current/_includes/cockroachcloud/create-cluster-api-dedicated.md

```diff
@@ -0,0 +1,96 @@
+**API request**
+
+To create a **Dedicated** cluster, send a `POST` request to the `/v1/clusters` endpoint, specifying the following parameters:
+
+- `name`: your cluster's name, a short string with no whitespace.
```

Comment:
```suggestion
- `name`: Your cluster's name, a short string with no whitespace.
```

Reply:
The following on these parameters are all nits, feel free to ignore as you want.